sdk/tests/invariant.test.ts

```diff
@@ -248,4 +248,198 @@ describe('invariant', async () => {
     const pools = await invariant.getPools(account)
     assert.deepEqual(pools.length, 1)
   })
+  it('create position', async () => {
+    if (!token0.contract?.address || !token1.contract?.address || !invariant.contract?.address) {
+      throw new Error()
+    }
+
+    const feeTier = newFeeTier({ v: 10000000000n }, 1)
+    await invariant.addFeeTier(account, feeTier)
+
+    await invariant.createPool(
+      account,
+      token0.contract.address.toString(),
+      token1.contract.address.toString(),
+      feeTier,
+      { v: 1000000000000000000000000n },
+      0n
+    )
+
+    await token0.approve(account, invariant.contract.address.toString(), 1000000000n)
+    await token1.approve(account, invariant.contract.address.toString(), 1000000000n)
+
+    const poolKey = newPoolKey(
+      token0.contract.address.toString(),
+      token1.contract.address.toString(),
+      feeTier
+    )
+
+    const lowerTickIndex = -10n
+    const upperTickIndex = 10n
+
+    await invariant.createPosition(
+      account,
+      poolKey,
+      lowerTickIndex,
+      upperTickIndex,
+      { v: 1000000n },
+      { v: 0n },
+      { v: 100000000000000000000000000n }
+    )
+    const position = await invariant.getPosition(account, 0n)
+    assert.deepEqual(position, {
+      poolKey: position.poolKey,
+      liquidity: { v: 1000000n },
+      lowerTickIndex: lowerTickIndex,
+      upperTickIndex: upperTickIndex,
+      feeGrowthInsideX: { v: 0n },
+      feeGrowthInsideY: { v: 0n },
+      lastBlockNumber: position.lastBlockNumber,
```

Comment:
Same example of: `position.lastBlockNumber === position.lastBlockNumber`. It's impossible to predict the value of this field. I suggest defining test-specific utility file. Add there `positionEquals` function that compare fields as skip comparing fields like the one mentioned